src/Helpers/ContextHelper.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\BLT\Helpers;
+
+use Illuminate\Support\Str;
+
+class ContextHelper
+{
+    public static function getHelper(string $helper)
+    {
+        $helperClass = match($helper) {
+            "array" => config("blt.helpers.array"),
+            "boolean" => config("blt.helpers.boolean"),
+            "class" => config("blt.helpers.class"),
+            "context" => config("blt.helpers.context"),
+            "nullable" => config("blt.helpers.nullable"),
+        };
+
+        if (!$helperClass) {
+            $helperClass = "Blumilk\BLT\Helpers\\" . Str::studly($helper) . "Helper";
+        }
+
+        if (!class_exists($helperClass)) {
+            throw new \Exception("Helper class $helperClass does not exist.");
+        }
+
+        return new $helperClass();
```

Comment:
As far as I think about it, I think I prefer multiple method like `getUserHelper(): UserHelper` and so on. I don't see any case when someone would like to use non-existing helper here.